PICS/Resources/Localizable.xcstrings

```diff
@@ -174,6 +180,12 @@
           }
         }
       }
+    },
+    "Height" : {
```

Comment:
It might be great to avoid using texts directly but use localizable instead. E.g. define key-value pair in the localizable file, with the value to be the actual text. Then, you can use `String(localized: "<key for the localizable>")` or `LocalizedStringResource("<key for the localizable>")` instead of the texts directly. This makes it much easier if we want to switch to another language, such as German.